apps/src/templates/codeReviewGroups/CodeReviewGroupsManager.jsx

```diff
@@ -24,6 +24,28 @@ export default function CodeReviewGroupsManager({initialGroups}) {
   const getAssignedGroups = () =>
     groups.filter(group => group.droppableId !== DROPPABLE_ID_UNASSIGNED);
 
+  const onGroupNameUpdate = (droppableId, newName) => {
+    const updatedGroup = {...getGroup(droppableId), ...{name: newName}};
+    const updatedGroups = updateGroups(groups, [updatedGroup]);
+    setGroups(updatedGroups);
+  };
+
+  const onGroupDelete = droppableId => {
+    // First, take all group members from deleted group and put them in unassigned group
+    const updatedUnassignedGroup = unassignAllFromGroup(droppableId);
+    const updatedGroups = updateGroups(groups, [updatedUnassignedGroup]);
+
+    setGroups(updatedGroups.filter(group => group.droppableId !== droppableId));
+  };
+
+  const unassignAllFromGroup = droppableId => {
+    const updatedUnassignedGroup = {...getUnassignedGroup()};
```

Comment:
should this method also delete the members from the original group? Not necessary for the delete scenario, but it will be necessary for just unassigning members from a group without deleting the group

Reply:
mmm yes I think so. I'm doing the unassign button as a follow-up, I'll add this logic there if that's ok.